src/main/java/seedu/address/logic/commands/AddGroupCommand.java

```diff
@@ -20,16 +21,17 @@
 public class AddGroupCommand extends Command {
     public static final String COMMAND_WORD = "add-group";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds the group of the specified name with the"
-            + "specified persons indicated by the index numbers used in the last person listing. If group exists,"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds the group of the specified name with the "
+            + "specified persons indicated by the index numbers used in the last person listing. If group exists, "
             + "add people into group if they are not in it.  \n"
             + "Parameters: " + PREFIX_NAME + "GROUP_NAME (must be alphanumeric only) "
             + PREFIX_PERSONS + "[INDEX]\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "Close Friends" + " "
             + PREFIX_PERSONS + "1 2 3 4";
 
-    public static final String MESSAGE_ADD_GROUP_SUCCESS = "Added %1$s into group %2$s";
+    public static final String MESSAGE_ADD_GROUP_SUCCESS = "Added into group %1$s";
+    public static final String MESSAGE_ADD_DEFAULT_GROUP_ERROR = "Unable to add to the Default Group, %1$s.";
```

Comment:
Can reword the error message to `"There is no need to add people to \"%1$s\" as everyone is included by default."`